null_test.go

```diff
@@ -205,10 +205,10 @@ func TestNullUUIDUnmarshalJSON(t *testing.T) {
 	var nu NullUUID
 	err := json.Unmarshal(jsonNull, &nu)
 	if err != nil || nu.Valid {
-		t.Errorf("expected nil when unmarshaling null, got %s", err)
+		t.Errorf("expected nil when unmarshalilng null, got %s", err)
```

Comment:
There are no typos in `unmarshaling`. See  https://github.com/golang/go/wiki/Spelling